src/Summoner/CLI.hs

```diff
@@ -41,6 +47,35 @@ summon = execParser prsr >>= runCommand
 runCommand :: Command -> IO ()
 runCommand = \case
     New opts -> runNew opts
+    ShowInfo opts -> runShow opts
+
+runShow :: ShowOpts -> IO ()
+runShow opts =
+    case opts of
+        -- show list of all available GHC versions
+        ShowGhc          -> putText $ unlines $ map showGhcVer universe
+        ShowLicense name -> case name of
+            -- show a list of all available licenses
+            LicenseList Nothing  -> putText $ unlines $ map (\x -> show (x :: LicenseName)) universe
+            -- show a specific license
+            LicenseList (Just n) -> do
+                -- check a user`s input
+                license <- whenNothing (parseLicenseName (toText n))
+                    (fail "This wasn't a valid choice.")
+                -- get a link to github
+                let licenseLink = "https://api.github.com/licenses/" <> githubLicenseQueryNames license
+                -- get a JSON of specific license from github
+                licenseJson <-
+                    readProcess "curl"
+                                [ toString licenseLink
+                                , "-H"
+                                , "Accept: application/vnd.github.drax-preview+json"
+                                ]
+                                ""
+                -- get a license`s text and show
+                case (decodeStrict $ pack licenseJson) :: Maybe License of
+                    Just t  -> putStr $ unLicense t
+                    Nothing -> error "Broken predefined license list"
```

Comment:
This chunk of code is actually absolutely the same as the one user in `Project` module. So it makes sense to put it into `License` module and use in both cases.